src/status_im2/contexts/syncing/setup_syncing/style.cljs

```diff
@@ -1,5 +1,6 @@
 (ns status-im2.contexts.syncing.setup-syncing.style
-  (:require [quo2.foundations.colors :as colors]))
+  (:require [quo2.foundations.colors :as colors]
+            [status-im2.contexts.syncing.standard-authentication.view :as standard-auth]))
```

Comment:
looks like this `require` - "[status-im2.contexts.syncing.standard-authentication.view :as standard-auth]" is not needed